bar-trender-front-end/src/views/ListView.js

```diff
@@ -87,32 +56,33 @@ function ListPage() {
       if (data['Triana'] == "on") {
         zones_aux.push("Triana");
       }
-  
-      filter = {
-  
-        "filters": {
-          "zones": zones_aux,
-          "beers": beers_aux,
-    
+
+        if(zones_aux.length != 0){
+          filter["filters"]["zones"] = zones_aux;
+        }
+        if(beers_aux.length != 0){
+          filter["filters"]["beers"] = beers_aux;
         }
-      };
-  
+      
     }
-    
 
     const apiUrl = "https://develop-backend-sprint-01.herokuapp.com/v1/establishments/get";
 
-    fetch(apiUrl, {
-      method: 'POST',
-      body: JSON.stringify(filter),
-      headers: {
-        'Content-Type': 'application/json'
-      }
-    })
-      .then(response => response.json())
-      .then(establishments => {
-        setAppState({ loading: false, establishments: establishments });
-      });
+    async function loadResults() {
+      await fetch(apiUrl, {
+        method: 'POST',
+        body: JSON.stringify(filter),
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      })
+        .then(response => response.json())
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7163535)
```suggestion
        .then((response) => response.json())
```